cli/tools/test.rs

```diff
@@ -1004,48 +1004,47 @@ async fn test_specifiers(
   };
 
   let (sender, mut receiver) = unbounded_channel::<TestEvent>();
-  let fail_fast_tracker = FailFastTracker::new(options.fail_fast);
   let sender = TestEventSender::new(sender);
   let concurrent_jobs = options.concurrent_jobs;
 
   let join_handles =
-    specifiers_with_mode.iter().map(move |(specifier, mode)| {
-      let ps = ps.clone();
-      let permissions = permissions.clone();
-      let specifier = specifier.clone();
-      let mode = mode.clone();
-      let mut sender = sender.clone();
-      let options = options.clone();
-      let fail_fast_tracker = fail_fast_tracker.clone();
-
-      tokio::task::spawn_blocking(move || {
-        if fail_fast_tracker.should_stop() {
-          return Ok(());
-        }
+    specifiers_with_mode
+      .into_iter()
+      .map(move |(specifier, mode)| {
+        let ps = ps.clone();
+        let permissions = permissions.clone();
+        let mut sender = sender.clone();
+        let options = options.clone();
+        let fail_fast_tracker = FailFastTracker::new(options.fail_fast);
```

Comment:
this seems like unnecessary indirection for no real benefit

Reply:
The tracker is not used anywhere else so I moved it inside the closure to avoid a clone